kubernetes/helm_charts/core/adminutils/values.j2

```diff
@@ -22,6 +22,27 @@ strategy:
   maxsurge: 1
   maxunavailable: 0
 
+adminutils:
+  readinessprobe:
+    enabled: true
+  livenessprobe:
+    enabled: true 
+  readinessProbe:
+    httpGet:
+      path: /health
+      port: 4000
+    initialDelaySeconds: 30
```

Comment:
Variablize the timeout values with safe defaults for each service with a provision to override in private repo

Reply:
even for a single path overriding we've to override, entire dictionary. So I think variablizing only delaySeconds will add more variables.